fish_prompt.fish

```diff
@@ -12,6 +17,6 @@ function fish_prompt
 	__sf_section_dir cyan $brwhite magenta red
 	__sf_section_exec_time $brwhite yellow
 	echo
-	echo -e -n -s (set_color $arrow_color) "➜ "
+	echo -e -n -s (set_color $arrow_color) "$SPACEFISH_CHAR_PREFIX$SPACEFISH_CHAR_SYMBOL$SPACEFISH_CHAR_SUFFIX"
```

Comment:
I would now consider moving the arrow prompt into a separate "char section" fish function, like I have for the other sections to parallel the [char section](https://github.com/denysdovhan/spaceship-prompt/blob/master/sections/char.zsh) in spaceship.

Feel free to create a new fish function for that, otherwise I'll move this stuff over in the next update.